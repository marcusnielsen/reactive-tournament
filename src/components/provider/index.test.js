import Provider from '../provider'

test('create', () => {
  const provider = Provider()
  const actual = Object.keys(provider)
  expect(actual).toMatchSnapshot()
})
