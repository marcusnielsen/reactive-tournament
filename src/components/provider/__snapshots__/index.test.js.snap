exports[`test create 1`] = `
Array [
  "localStorage",
  "entityServices",
  "schemas",
  "colors",
]
`;
